index.html

```diff
@@ -8,33 +8,179 @@
     <link rel="stylesheet" href="style.css">
 </head>
 <body>
-   <header class="header-container">
-    <nav class="nav">
-        <a>My logo</a>
-        <img src="images/hamburger.png" alt="">
-    </nav>
-    <section class="headline">
-        <a  class="greetings">I’m Ignatius Glad to see you!</a>
-        <p class="welcome-message">I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
-        <a class="heading-1">LET’S CONNECT</a>
-        <ul class="icon-bar">
-            <li>
-                <img src="images/twitter.png" alt="">
-            </li>
-            <li>
-                <img src="images/Linkedin icon.png" alt="">
-            </li>
-            <li>
-                <img src="images/m.png" alt="">
-            </li>
-            <li>
-                <img src="images/cat.png" alt="">
-            </li>
-            <li>
-                <img src="images/hand.png" alt="">
-            </li>
-        </ul>
-    </section>
-   </header>
+  <section class="header-container">
+    <header>
+        <nav class="nav">
+            <a>My logo</a>
+            <img src="images/hamburger.png" alt="">
+        </nav>
+        <section class="headline">
+            <a  class="greetings">I’m Ignatius Glad to see you!</a>
+            <p class="welcome-message">I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+            <a class="heading-1">LET’S CONNECT</a>
+            <ul class="icon-bar">
+                <li>
+                    <img src="images/twitter.png" alt="">
+                </li>
+                <li>
+                    <img src="images/Linkedin icon.png" alt="">
+                </li>
+                <li>
+                    <img src="images/m.png" alt="">
+                </li>
+                <li>
+                    <img src="images/cat.png" alt="">
+                </li>
+                <li>
+                    <img src="images/hand.png" alt="">
+                </li>
+            </ul>
+        </section>
+       </header>
+       <main>
+        <section class="work" >
+            <div class="tonic-1">
+                <img src="images/tonic.png" alt="">
+                <p class="tonic">Tonic</p>
```

Comment:
- [ ] While you have done a great job, this p tag is not meant to be there, it is required to use an (h2....h5) tag to get the project title, kindly fix it.